### ISSUE TITLE
[Bug] Anonymous cheers have $username set to the broadcaster name

### ISSUE THREAD

**Issue by mgway** [crowbartools/Firebot#1705]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

If someone cheers anonymously, the `$user` and `$username` variables will be set to the broadcaster name, not blank or "AnAnonymousCheerer". This issue is not apparent in event simulation due to the manually entered username being minimally set to `""`, not `null|undefined`

Having a peek at the code, it looks like this is caused by `twurple PubSubBitsMessage` having `userName: undefined` for anonymous cheers which triggers the fallback logic in `EventHander.triggerEvent`
```
if (meta.username == null) {
     const accountAccess = require("../common/account-access");
     meta.username = accountAccess.getAccounts().streamer.username;
}
```

### Expected Behavior

The $user of an anonymous cheer is "An Anonymous Cheerer" or simply "AnAnonymousCheerer"

### Steps To Reproduce

1. Set up a new cheer event
2. Set the effect to a chat message with text `$username has cheered`
3. Cheer any amount in the channel.  This issue is _not_ triggered via simulating events, the cheer event must come in via a Twitch pubsub message

### Version

0.51

### What operating system are you using Firebot on?

Windows 10

### Contact Details (Optional)

oh_mg#1234

**Comment by ebiggz** [comment]
Thanks for the pull request for this @mgway!

**Comment by SReject** [comment]
Resolved with v5.52.0
